### PR TITLE
Am/feature add tags for topic

### PR DESCRIPTION
**Pull request by 11001**

### CODE REVIEW

**Comment by binaryvo** [comment]
Please update from next, resolve conflicts and re-open PR

File: resources/assets/js/app/views/votes/CreateVote.js [hunk 1]
```diff
@@ -117,27 +130,27 @@ module.exports = Marionette.LayoutView.extend({
...
-            });
-        }
         view.model.save({
             users: JSON.stringify(users),
-            tags: JSON.stringify(tags),
+            tags: this.ui.tagsInput.val(),
             is_saved: 1
         }, {
             success: function (data) {
                 Backbone.history.navigate('votes/' + data.get('id'), {trigger: true});
```
Comment (binaryvo) [inline resources/assets/js/app/views/votes/CreateVote.js]: Please use the slug in url

**Comment by binaryvo** [comment]
I have an issue with list of existing topics. It looks not so good
selection_005
Probably it should be below field of tags

File: resources/assets/js/app/views/votes/CreateVote.js [hunk 2]
```diff
@@ -114,7 +126,8 @@ module.exports = Marionette.LayoutView.extend({
             collection: this.getOption('accessedUsers'),
             childView: require('./CreateVoteUserItemExtend')
         }));
-    },
+    }
+    ,
```
Comment (binaryvo) [inline resources/assets/js/app/views/votes/CreateVote.js]: Please move coma after closing brace

File: resources/assets/js/app/views/votes/CreateVote.js [hunk 3]
```diff
@@ -126,27 +139,27 @@ module.exports = Marionette.LayoutView.extend({
...
+            tags: this.ui.tagsInput.val(),
             is_saved: 1
         }, {
             success: function (data) {
                 Backbone.history.navigate('votes/' + data.get('slug'), {trigger: true});
             }
         });
-    },
+    }
+    ,
```
Comment (binaryvo) [inline resources/assets/js/app/views/votes/CreateVote.js]: Please move coma after closing brace.

File: resources/assets/js/app/views/votes/CreateVote.js [hunk 4]
```diff
@@ -126,27 +139,27 @@ module.exports = Marionette.LayoutView.extend({
...
     saveModel: function (obj) {
         if (this.model.get('id')) {
             this.model.save(obj);
         } else {
             this.model.set(obj);
         }
     }
-});
\ No newline at end of file
+    ,
```
Comment (binaryvo) [inline resources/assets/js/app/views/votes/CreateVote.js]: Please move coma after closing brace.

File: resources/assets/js/app/views/votes/voteHeader.js [hunk 5]
```diff
@@ -47,7 +47,7 @@ module.exports = Marionette.ItemView.extend({
                 showUsers: currentUser.isAdmin() || (currentUser.get('id') === this.model.get('user_id'))
             }
         }
-
+        console.log(meta);
```
Comment (binaryvo) [inline resources/assets/js/app/views/votes/voteHeader.js]: Please remove or comment consloe.log()

File: resources/assets/js/app/instances/Model.js [hunk 6]
```diff
@@ -27,6 +27,7 @@ module.exports = Backbone.Model.extend({
         options.statusCode['400'] = function (xhr, textStatus, errorThrown) {
             if (xhr.responseJSON) {
                 model.validationError = xhr.responseJSON;
+                console.log(xhr.responseJSON);
```
Comment (binaryvo) [inline resources/assets/js/app/instances/Model.js]: Please remove or comment console.log()

**Comment by binaryvo** [comment]
Please fix all remarks and re-open PR

**Comment by binaryvo** [comment]
Strange behavior of plugin for tags. Sometimes tag doesn't add to list. [Video](http://www.ex.ua/751062838419)

**Comment by 11001** [comment]
You're trying to add duplicate tags. It may be strange.

**Pull request merged**